psamm/commands/table.py

```diff
@@ -0,0 +1,179 @@
+
+
+
+from ..command import Command
+import logging
+import xlsxwriter
+
+logger = logging.getLogger(__name__)
+class ExportTableCommand(Command):
+    """Export parts of the model as tab separated tables"""
+
+
+    @classmethod
+    def init_parser(cls, parser):
+        parser.add_argument(
+            'export', metavar='export_type', help='reactions, compounds, medium, limits')
+
+    def run(self):
+        model = self._model
+        export_type = self._args.export
+        if export_type == 'reactions':
+            reaction_export(model)
+        if export_type == 'compounds':
+            compound_parser(model)
+        if export_type == 'medium':
+            media_parser(model)
+        if export_type == 'limits':
+            limits_parser(model)
+        if export_type == 'metadata':
+            metadata_parser(model)
+        if export_type == 'xlsx':
+            xlsx_writer(model)
+
+
```

Comment:
I would call the functions "xxx_writer" or "xxx_export" (pick one for consistency) as you did with `reaction_export` and `xlsx_writer`.